src/workerd/io/worker-fs-test.c++

```diff
@@ -0,0 +1,120 @@
+#include "worker-fs.h"
+
+#include <kj/debug.h>
+#include <kj/test.h>
+
+namespace workerd {
```

Comment:
nit: using `workerd::` everywhere in this file even though you are in the `workerd` namespace